addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -164,6 +208,28 @@ export class HtmlField extends Component {
             },
         };
     }
+    /**
+     * Prevent usage of the dynamic placeholder command inside widgets
+     * containing background images ( cover & masonry ).
+     *
+     * We cannot use dynamic placeholder in block containing background images
+     * because the email processing will flatten the text into the background
+     * image and this case the dynamic placeholder cannot be dynamic anymore.
+     *
+     * @param {Array} commands commands available in this wysiwyg
+     * @returns {Array} commands which can be used after the filter was applied
+     */
+    _filterPowerBoxCommands(commands) {
```

Comment:
I suppose this is related to the fact that now there is a dynamic placeholder in the email templates ? If yes then I suggest moving the set of changes relevant to this particular change into its own commit.